packages/server/src/db/records/badge.ts

```diff
@@ -14,9 +14,8 @@ import { UserDid } from '../user-dids'
 import schemas from '../schemas'
 import { collectionToTableName } from '../util'
 
-const schemaId = 'blueskyweb.xyz:Badge'
-const collection = 'bsky/badges'
-const tableName = collectionToTableName(collection)
+const type = 'todo.social.badge'
```

Comment:
this is a nit, but I kinda want a better word here than `type`. maybe `collection`? even `schemaType` or `collectionType`?

Reply:
butttt im very glad to have united the two diff concepts of `colleciton` & `schemaId` & I like the NSID syntax better than both of the previous versions

Reply:
I currently think of it as "record type" and "type" is just short for that, though I also think of "collection" as being equivalent so in this case I'd be fine with `collection`